bin/run-ansible.sh

```diff
@@ -80,6 +80,8 @@ echo "inventory    = ${aurora_inventory}"
 echo "limit        = ${aurora_limit}"
 
 export ANSIBLE_ROLES_PATH="${aurora_home}/ansible/roles"
+export ANSIBLE_CALLBACK_PLUGINS="~/.ansible/plugins/callback:/usr/share/ansible/plugins/callback:${aurora_home}/ansible/playbooks/callback_plugins"
```

Comment:
We should try to avoid using ~, and specify folders explicitly, e.g. /home/$USER/.ansible/plugins/callback